channeldb/graph.go

```diff
@@ -1529,16 +1529,29 @@ func (c *ChannelGraph) FilterKnownChanIDs(chanIDs []uint64) ([]uint64, error) {
 		if edgeIndex == nil {
 			return ErrGraphNoEdgesFound
 		}
+		zombieIndex := edges.Bucket(zombieBucket)
+		if edgeIndex == nil {
```

Comment:
Should be checking `zombieIndex` here. Also we don't want to return an error if it doesn't exist, as it's possible that a node hasn't marked any edges as zombies and so the bucket won't exist.

Reply:
heh good catch. we should really consider having migrations to initialize new buckets, or always initializing all top-level buckets on start up...